Assets/Scripts/Traps/IInteract.cs

```diff
@@ -0,0 +1,8 @@
+public interface Interact
```

Comment:
I found appropriate to add an `IInteract` interface to apply to all interactable objects. However, it may not be necessary at the moment, since there's only one of them.